src/decorator-annotator.ts

```diff
@@ -130,6 +133,26 @@ export class DecoratorClassVisitor {
     this.propDecorators.set(name, decorators);
   }
 
+  private getValueIdentifierForType(typeSymbol: ts.Symbol): ts.Identifier|null {
```

Comment:
Can you add docs on this?  What is a value identifier (or value declaration)? Something about literal types?